.github/scripts/pack-toolchain.sh

```diff
@@ -3,8 +3,13 @@
 source `dirname $0`/config.sh
 
 echo "::group::Pack toolchain"
-mkdir -p $ARTIFACT_PATH
-tar czf $ARTIFACT_PATH/$TOOLCHAIN_PACKAGE_NAME -C $TOOLCHAIN_PATH .
+    mkdir -p $ARTIFACT_PATH
+    if [[ $TOOLCHAIN_NAME == "aarch64-w64-mingw32-msvcrt" ]]; then
+        tar czf $ARTIFACT_PATH/$TOOLCHAIN_PACKAGE_NAME -C $TOOLCHAIN_PATH .
+    else
+        tar czf $ARTIFACT_PATH/$TOOLCHAIN_PACKAGE_NAME --files-from /dev/null
```

Comment:
is empty tar.gz really needed?

Reply:
maybe workflow check is a better place for this change

Reply:
> is empty tar.gz really needed?

At least something needs to be cached, otherwise it would produce cache miss.

Reply:
> maybe workflow check is a better place for this change

Which of the following do you have in mind:

1. Move the decision to the workflow passing "create empty archive" to this condition as a script parameter?
2. Move the decision to the workflow always calling one of two spteps  ("pack toolchain", and "pack empty toolcian") while the other will be skipped?

Reply:
the second option looks good for me. I would use other naming for "pack empty toolchain", something like "cache anchor" or some other options